generator/src/test/java/com/faforever/neroxis/generator/MapGeneratorTest.java

```diff
@@ -98,6 +99,14 @@ public void TestParseMapName() {
         assertEquals(generatorParameters.mapSize(), mapSize);
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(ValidTerrainAndMapSizeArgumentProvider.class)
+    public void TestEqualityTerrainAndMapSizeGeneratorSpecified(TerrainStyle terrainStyle, int mapSize) {
+        instance = new MapGenerator();
+
+        new CommandLine(instance).execute("--terrain-style", terrainStyle.toString(), "--map-size", String.valueOf(mapSize));
+    }
```

Comment:
This test doesn't test equality for determinism so should just be renamed to like test terrain size compatibility or something